internal/transactionsubmission/wallet_creation_transaction_handler.go

```diff
@@ -194,17 +192,6 @@ func (h *WalletCreationTransactionHandler) BuildInnerTransaction(ctx context.Con
 		return nil, applyErr
 	}
 
-	txParams.BaseFee = h.calculateAdjustedBaseFee(simulationResponse)
-
```

Comment:
The fee adjustment was a workaround needed due to a design restriction on the wallet-backend repo. I removed the call because it's not needed for us here.